webplugin/js/app/km-widget-custom-events.js

```diff
@@ -0,0 +1,100 @@
+/**
+ // Kommunicate custome events available for user for the specific event in chat widget.
+ // resp - JSON object containing eventAction and other property exposed to user.
+ // defination of function can be overwrite by user through subscribeToEvents function.
+ */
```

Comment:
create event mapping here
  var eventMapping = { 'FAQ Menu': {eventCateogry: 'Kommunicate',
                eventAction: 'Click',
                eventLabel: 'FAQ Menu'} ,
                // ... add same for all events...
                
                };

Reply:
Rectified